x/stablestake/keeper/debt.go

```diff
@@ -79,7 +79,7 @@ func (k Keeper) GetAllDebts(ctx sdk.Context) []types.Debt {
 }
 
 func (k Keeper) SetInterestForPool(ctx sdk.Context, poolId uint64, block uint64, interest types.InterestBlock) {
-	store := prefix.NewStore(runtime.KVStoreAdapter(k.storeService.OpenKVStore(ctx)), types.GetInterestKey(poolId))
+	store := prefix.NewStore(runtime.KVStoreAdapter(k.storeService.OpenKVStore(ctx)), types.GetNewInterestKey(poolId))
```

Comment:
Why do we need to new interest key? Can't we just delete all old keys and then use same?

Reply:
we did that earlier, I think there is some key collision, couldn't the reason why total value increased otherwise